pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md

```diff
@@ -0,0 +1,126 @@
+---
+title: AI Deploy - Apps portfolio
+slug: deploy/apps-portfolio
+excerpt: A collection of AI apps that can be easily deployed with AI Deploy
+section: AI Deploy guides
+order: 05
+---
+
+**Last updated 4th November, 2022.**
+
+> [!primary]
+>
+> AI Deploy is in `beta`. During the beta-testing phase, the infrastructure’s availability and data longevity are not guaranteed. Please do not use this service for applications that are in production, as this phase is not complete.
+>
+> AI Deploy is covered by **[OVHcloud Public Cloud Special Conditions](https://storage.gra.cloud.ovh.net/v1/AUTH_325716a587c64897acbef9a4a4726e38/contracts/d2a208c-Conditions_particulieres_OVH_Stack-WE-9.0.pdf)**.
+>
+
+## Objective
+
+AI Deploy allows you to deploy AI apps or models. To test or use the product, you can build on existing AI models.
```

Comment:
It's just a proposition: as we have _apps_ option in our CLI and that was the old name of _AI Deploy_, I think it's better to call it _applications_ instead _apps_

```suggestion
AI Deploy allows you to deploy AI applications or models. To test or use the product, you can build on existing AI models.
```